### PR TITLE
Docker java debug

### PR DESCRIPTION
**Pull request by madeye-matt**
- Added JAVA_OPTS definitions for all the containers running Java services
- Converted all environment variables from part1.part to PART1_PART2

### CODE REVIEW

**Comment by jwgwalton** [comment]
need to turn off travis on this repo

File: .env [hunk 1]
```diff
@@ -13,21 +13,27 @@ POSTGRES_PORT=5432
 # RM Services
 CASE_HOST=casesvc
 CASE_PORT=8171
+CASE_DEBUG_PORT=5171
```
Comment (benjefferies) [inline .env:16]: I wonder if we need any of these in the .env file or would it be easier to keep in the compose file?

**Review by benjefferies — CHANGES_REQUESTED** [review]

**Review by benjefferies — APPROVED** [review]

File: .env (hunk 1 above)
Comment (benjefferies) [reply .env:16]: We discussed this and we should look at it again when refactoring the compose file

Comment (benjefferies) [reply .env:16]: We discussed this and we should look at it again when refactoring the compose file

**Review by jwgwalton — APPROVED** [review]

**Pull request merged**